source/sync/production-checklist.txt

```diff
@@ -0,0 +1,66 @@
+.. _realm-sync-best-practices:
+.. _sync-production-checklist:
+
+================================
+Device Sync Production Checklist
```

Comment:
Since we're calling this a production checklist, I wonder if we want to consolidate the [production load testing page](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/sync-landing/sync/production-load-testing/) onto this page?

Reply:
I was thinking we could move both into a subsection called "Go to Production"?

Reply:
Works for me!